src/mcnp/mcnp_funcs.cpp

```diff
@@ -317,16 +318,19 @@ void write_surface_cards(std::ostringstream& lcadfile) {
 
 // write out all the tally data from the uwuw file
 void write_material_data(std::ostringstream& lcadfile) {
-  std::map<std::string, pyne::Material> material_library = workflow_data->material_library;
+  pyne::MaterialLibrary material_library = workflow_data->material_library;
```

Comment:
create a reference rather than a copy?

Reply:
ping @bam241

Reply:
I'd make that reference `const` as well.